library/src/interactor_impl.cxx

```diff
@@ -196,145 +197,51 @@ class interactor_impl::internals
   //----------------------------------------------------------------------------
   static void OnKeyPress(vtkObject*, unsigned long, void* clientData, void*)
   {
-
     internals* self = static_cast<internals*>(clientData);
     vtkRenderWindowInteractor* rwi = self->Style->GetInteractor();
-    int keyCode = std::toupper(rwi->GetKeyCode());
-    std::string keySym = rwi->GetKeySym();
-    if (keySym.length() > 0)
+    InteractionBind bind;
+    bind.Interaction = rwi->GetKeySym();
+    if (bind.Interaction.length() > 0)
     {
-      // Make sure key symbols starts with an upper char (e.g. "space")
-      keySym[0] = std::toupper(keySym[0]);
+      // Make sure key symbols starts with an upper char (e.g. "space" -> "Space")
+      bind.Interaction[0] = std::toupper(bind.Interaction[0]);
     }
 
-    if (self->KeyPressUserCallBack(keyCode, keySym))
+    // Check for an interaction command with modifiers
+    // XXX: Cannot use binary stuff here ?
+    bind.Modifiers = ModifierKeys::NONE;
+    int shift = rwi->GetShiftKey();
+    int ctrl = rwi->GetControlKey();
+    if (shift == 1 && ctrl == 1)
```

Comment:
```suggestion
    const bool shift = rwi->GetShiftKey() == 1;
    const bool ctrl = rwi->GetControlKey() == 1;
    if (shift && ctrl)
```
because only `==1` is checked (can they even be anything than `0` or `1`?)